<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_profile">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>511</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D8BFD8</string>
    </property>
    <property name="text">
     <string>Профиль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_address">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080</string>
    </property>
    <property name="text">
     <string>Адреса книжных полок</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_find">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>210</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D8BFD8</string>
    </property>
    <property name="text">
     <string>Взять книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_share">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>330</y>
      <width>511</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080</string>
    </property>
    <property name="text">
     <string>Поделиться книгой</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
